### ISSUE TITLE
Terminates when run in a loop (Linux)

### ISSUE THREAD

**Issue by phaggy** [OzymandiasTheGreat/klembord#10]
program terminates with this error message when run in a loop:
``` Xlib.error.DisplayConnectionError: Can't connect to display ":0": b'Maximum number of clients reached' ```

Here is the snippet:
```
while True:
        klembord.init()
        print(klembord.get_text())
```

**Comment by OzymandiasTheGreat** [comment]
Just as the error says, you're making too many connections to X server. There's some basic cleanup, but you're racing it (and winning!).
klembord is designed to be used stand alone independent of any toolkits. There's no event loop, so we can't maintain a single connection to the server. I tried to mitigate this with background threads, but xlib doesn't play very nice with threading, so we can't use single connection from multiple threads.
Basically, try not to use klembord in an infinite loop or add delays.
